README.rst

```diff
@@ -190,16 +190,23 @@ Tests
 This library includes a sample testing utility which may be used to verify
 that changes to the processor maintain the correct output.
 
-To run the sample tests you will need to get the test suite files by cloning
-the ``json-ld-api``, ``json-ld-framing``, and ``normalization`` repositories
-hosted on GitHub:
+To run the sample tests you will need the test suite files provided in the ``json-ld-api``,
```

Comment:
Please line wrap this and below changes at 80 columns.

Reply:
Fixed :white_check_mark: